sentry_dart_plugin/lib/src/utils/extensions.dart

```diff
@@ -0,0 +1,3 @@
+extension StringValidations on String? {
+  bool get isNull => this == null;
```

Comment:
@bruno-garcia, you might say we should also add `isNullOrEmpty` 😁. @marandaneto, I'm just kidding. [`isNullOrEmpty`](https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorempty) is a common C# thing.